Blocks/URL/test.cc

```diff
@@ -74,7 +74,8 @@ TEST(URLTest, SmokeTest) {
   EXPECT_EQ(8080, u.port);
   EXPECT_EQ("", u.username);
   EXPECT_EQ("", u.password);
-  // NOTE: `ComposeURL` renders default scheme if scheme is empty, not known by port, but host is present (i.e. absolute URLs).
+  // NOTE: `ComposeURL` renders default scheme if scheme is empty, not known by port, but host is present (i.e. absolute
+  // URLs).
```

Comment:
Note them. Or merge this PR and send a new one my way. Or push a commit fixing the indent -- in a way that doesn't contradict `clang-format-3.6` -- into my branch.